src/igvfd/schemas/changelogs/reference_file.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`reference_file.json`*
 
+### Schema version 4
+
+* Rename enums in `transcriptome_annotation` property by adding `GENCODE` prefix to `M30`, `M31`, `M32`, `V40`, `V41`, `V42`, and `V43`.
```

Comment:
there is no enum that reads: GENCODE V41 as an example. Please drop the detail of which entries gained GENCODE prefix. Instead you can say: "* Update enums in `transcriptome_annotation` to include `GENCODE` prefix."

Reply:
done